kit/popups/src/useTargetDimensions.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import type { NativeMethods, ScaledSize } from 'react-native';
+
+export type TargetDimensions = {
+    x: number;
+    y: number;
+    width: number;
+    height: number;
+};
+
+export function useTargetDimensions(
+    targetRef: React.RefObject<NativeMethods>,
+    windowDimensions: ScaledSize,
+): TargetDimensions | null {
+    const [targetDimensions, setTargetDimensions] = React.useState<TargetDimensions | null>(null);
+
+    React.useEffect(() => {
+        if (!targetRef.current || !targetRef.current.measure) {
+            console.error(
+                `[UITooltipContent]: [useTargetDimensions]: targetRef does not have a 'measure' method`,
+            );
+            return;
+        }
+        targetRef.current.measure(
```

Comment:
Shouldn't it be `measureInWindow`? 🤔 or is it because of `Portal` that takes whole available space?

Reply:
I agree. I think it will be more correct to use `measureInWindow`